CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 Mapbox welcomes participation and contributions from everyone.
 # main
+
+## Bug fixes 🐞
+* Preserve cached layer properties during the getLayer call. ([1622](https://github.com/mapbox/mapbox-maps-android/pull/1622))
```

Comment:
based on the delta here this PR was not rebased on latest main for quite a while, could you please do that?